packages/provider/src/language-model/v1/language-model-v1.ts

```diff
@@ -125,6 +131,9 @@ export type LanguageModelV1StreamPart =
   // Basic text deltas:
   | { type: 'text-delta'; textDelta: string }
 
+  // Logprob results:
+  | { type: 'log-probs'; logprobs: LanguageModelV1LogProbs }
+
```

Comment:
Instead of introducing a separate stream part, please add a `logprobs` property to the `finish` part (and also change that behavior in streamText etc).

Reply:
Done! I found that streaming objects only provides a `partialObjectStream`. I've quite heavily modified the `stream-object.ts` file to also provide a `fullStream` property. It works exactly the same as the `fullStream` from `streamText`, except it provides the aggregated object under the `object` type. Example:

```typescript
const result = await experimental_streamObject({
    model: openai.chat('gpt-3.5-turbo', { logprobs: 1 }),
    maxTokens: 2000,
    schema: z.object({ ... }); // type T
})
for await (const part of result.fullStream) {
    switch (part.type) {
      case 'object':
        console.clear();
        console.log(part.object); // of Type T
        break;
      case 'finish':
        console.log('logprobs', part.logprobs);
        break;
    }
  }
  ```

Reply:
this is a great change, wanted to do something in this direction anyways!